### PR TITLE
feat: support multiple languages and add Chinese translation

### PR DESCRIPTION
**Pull request by misitebao**
It mainly solves the problems mentioned in [Help site supports multiple languages](https://github.com/go-task/task/issues/1057), and the specific work is as follows:

- [x] Add multilingual capabilities to the documentation site
- [x] Add Crowdin CLI tool and workflow
- [x] Add Simplified Chinese translation (60%)

Resolve: https://github.com/go-task/task/issues/1057

Preview: https://misitebao-taskfile-git-feature-1057-support-mu-5d0af6-misitebao.vercel.app/

### CODE REVIEW

File: docs/Taskfile.yml [hunk 1]
```diff
@@ -40,3 +40,18 @@ tasks:
     desc: Upgrade Docusaurus
     cmds:
       - yarn upgrade @docusaurus/core@latest @docusaurus/preset-classic@latest
+
```
Comment (andreynering) [inline docs/Taskfile.yml:43]:
```suggestion
```

File: docs/Taskfile.yml [hunk 2]
```diff
@@ -40,3 +40,18 @@ tasks:
...
+    cmds:
+      - npx crowdin push
+
+  crowdin:pull:
+    desc: Download latest translations from Crowdin to the specified place
+    deps: [yarn:install]
+    cmds:
+      - npx crowdin pull --export-only-approved
+
+
```
Comment (andreynering) [inline docs/Taskfile.yml:57]:
```suggestion
```

File: .github/workflows/sync-translated-documents.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+      - name: Install Task
+        uses: arduino/setup-task@v1
+        with:
+          version: 3.x
+
+      - name: Sync Translated Documents
+        run: task crowdin:pull
+        env:
+          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
```
Comment (andreynering) [inline .github/workflows/sync-translated-documents.yml:21]: Added 👍

File: docs/crowdin.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+  # - source: /src/pages/**/*
+  #   translation: /i18n/%locale%/docusaurus-plugin-content-pages/**/%original_file_name%
+  #   ignore:
+  #     - /**/*.js
+  #     - /**/*.jsx
+  #     - /**/*.ts
+  #     - /**/*.tsx
+  #     - /**/*.css
+
```
Comment (andreynering) [inline docs/crowdin.yml:22]: We can delete this if it won't be needed.

**Review by andreynering — COMMENTED** [review]
Hi @misitebao,

First of all, thanks for your work setting all this up! In general this seems to be working very well for me, and it's a good addition to the website!

Something I noticed: when running the Docusaurus server locally the translations are not being generated for some reason, and we get a 404 when trying to see any translated page. When running `task docs:build` everything works perfectly, though, so probably not a big deal for now.

Screenshot 2023-03-16 at 22 35 32

Let me know once you think you have finished and this is ready to be merged. I may do a few adjustments once in master.

**Comment by misitebao** [comment]
> Something I noticed: when running the Docusaurus server locally the translations are not being generated for some reason, and we get a 404 when trying to see any translated page. When running task docs:build everything works perfectly, though, so probably not a big deal for now.

This is because when using `docusaurus start`, you cannot start all language sites at the same time, if you want to access a specific language, please refer to the documentation: https://docusaurus.io/docs/i18n/tutorial#start-your-site

If you want to preview all languages at the same time, you can do this:
```
task build
task preview
```

**Comment by andreynering** [comment]
Thanks @misitebao!

**Pull request merged**